addons/remember-collapsed-categories/addon.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Remember collapsed forum categories",
+  "description": "Categories you collapsed on the forum home page (the list of forums) will stay collapsed when you leave and come back.",
```

The description can probably be improved, but I don't have any concrete suggestions. Low priority.

Some notes:
1. It should be clear that the collapsing feature is already available in vanilla Scratch, but that it doesn't persist tab sessions
2. Is /discuss/ called "the forum home page"? Do other addons name it differently?
3. "when you leave and come back" can probably be made clearer and less informal.

---

```suggestion
  "description": "Fixes a bug in Scratch where categories on the forum front page don't stay collapsed when reloading the page.",
```
It might not be specific enough though.

---

I don't think we need to indicate that it's a bug. Most Scratchers will think it simply wasn't a feature.

---

@DNin01 I agree. I'm not sure why I said "it should be clear that the collapsing feature is already available in vanilla Scratch, but that it doesn't persist tab sessions".

---

I think that I was trying to say that Scratch already offers the ability to collapse forum categories, and we shouldn't imply that we are the ones adding that. In other words, that the collapsing UI/UX already exists, but that it doesn't persist sessions by default.
But clearly, the user doesn't care that technically this fixes a bug.